hypervisor/network/network_linux.go

```diff
@@ -201,14 +201,16 @@ func UpAndAddToBridge(name, bridge, options string) error {
 	if bridge == "" {
 		bridge = BridgeIface
 	}
-	master, err := netlink.LinkByName(bridge)
-	if err != nil {
-		glog.Error("cannot find bridge interface ", bridge)
-		return err
-	}
-	if err = addToBridge(iface, master, options); err != nil {
-		glog.Errorf("cannot add %s to %s ", name, bridge)
-		return err
+	if bridge != "" {
```

Comment:
In which case BridgeIface is null? the bridge suppose not to be null here.

Reply:
From previous logic:
```
201     if bridge == "" {
202         bridge = BridgeIface
203     }
```

if we pass an empty `bridge`, it will fall down to `BridgeIface`, in my test, it's still empty which means BridgeIface is empty. 

I didn't look deeply into this, in my environment, I can see there's one new `runv0` bridge, and no `hyper0` bridge, not sure if this is related.